db/models.py

```diff
@@ -31,22 +34,80 @@
     rows = models.IntegerField()
     seats_in_row = models.IntegerField()
 
-    @property
-    def capacity(self) -> int:
-        return self.rows * self.seats_in_row
-
     def __str__(self) -> str:
         return self.name
 
 
 class MovieSession(models.Model):
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
     show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(
-        to=CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
-    )
-    movie = models.ForeignKey(
-        to=Movie, on_delete=models.CASCADE, related_name="movie_sessions"
+    cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        show_time_str = self.show_time.strftime('%Y-%m-%d %H:%M:%S')
+        return f"{self.movie.title} {show_time_str}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        get_user_model(),
```

Comment:
Code Style: You are using `get_user_model()` in the ForeignKey field which is a good practice, as per the checklist. However, you have also imported the `AbstractUser` model directly. It would be better to use `get_user_model()` everywhere instead of directly importing the user model.